README.md

```diff
@@ -5,6 +5,7 @@ PIConGPU - Particle-in-Cell Simulations for the Exascale Era
 [![Code Status dev](https://img.shields.io/travis/ComputationalRadiationPhysics/picongpu/dev.svg?label=dev)](https://travis-ci.org/ComputationalRadiationPhysics/picongpu/branches)
 [![Documentation Status](https://readthedocs.org/projects/picongpu/badge/?version=latest)](http://picongpu.readthedocs.io)
 [![Doxygen](https://img.shields.io/badge/API-Doxygen-blue.svg)](http://computationalradiationphysics.github.io/picongpu)
+[![GitHub commits since last release](https://img.shields.io/github/commits-since/ComputationalRadiationPhysics/picongpu/latest/dev.svg)](https://github.com/ComputationalRadiationPhysics/picongpu/compare/master...dev)
```

Comment:
Optional: add a comment, that loading this page might take same time due to the many commits

Reply:
Oops, perhaps I merged too soon, sorry.

Reply:
No, that's fine - was just optional.

Reply:
I would also not add a comment in the badge line, tbh